### PR TITLE
updated downlodes

### PR DESCRIPTION
**Pull request by sukar42**

### CODE REVIEW

**Comment by ZeroLevels** [comment]
The "Hosted On Github" link is supposed to point back to the Github repository in which someone can view all the files, submit pull request, etc... Not to lead back to itself.  Not only does leading back to itself serve no purpose, but it keeps people from more easily finding the place in where they can contribute.  I would recommend the links either being set back to whichever repository is currently being used instead of looping back to the gh-pages.

In other words, it should either link back to either https://github.com/sukar42/LAST_DAYS or https://github.com/ZeroLevels/LAST_DAYS .  Again, to whichever one is being used for main contributions to the pack.
